### DISCUSSION TITLE
Conditionals in lfrc?

### DISCUSSION THREAD

**Discussion by Lassebq** [gokcehan/lf#1404] [Q&A] [answered]
So I want to enable sixel only for terminals which support it, is there any way to do something like
```sh
if [[ "$TERM" = foot || -n "$KONSOLE_VERSION" ]]; then
    set sixel
fi
```
Is there any way I can do that?

**Comment by michaelferon** [comment, marked as answer]
I think the following would work in your `lfrc`:
```zsh
&{{
    if [[ "$TERM" = foot || -n "$KONSOLE_VERSION" ]]; then
        command lf -remote "send $id :set sixel"
    fi
}}
```
By not attaching the command to a `cmd` statement, it will run automatically at startup.

**Reply by michaelferon** [reply]
Search for 'REMOTE COMMANDS' in the man pages for more info about the implementation of control flow logic in `lf` in general

**Reply by Lassebq** [reply]
Thank you, but I already knew about remote commands, I just didn't know about functions without `cmd` statement running at startup
